src/qiskit_iqm/fake_backends/chip_samples/example_sample.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2022 Qiskit on IQM developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+"""
+    This module shows how to create a chip sample. Copy this file and populate it with your measured values.
+"""
+from .. import IQMChipSample
+from ..quantum_architectures import Adonis
+
+adonis_chip_sample = IQMChipSample(
+    quantum_architecture=Adonis(),  # can be Adonis() (5-qubit architecture) or Apollo() (20-qubit architecture)
+    # T1 times for all qubits, index indicates the qubit number
+    t1s={0: 50000.0, 1: 50000.0, 2: 50000.0, 3: 50000.0, 4: 50000.0},
+    # T2 times for all qubits, index indicates the qubit number
+    t2s={0: 50000.0, 1: 50000.0, 2: 50000.0, 3: 50000.0, 4: 50000.0},
+    # Gate fidelities for one and two qubit gates, the key in the dictionary indicates the qubits the gate operates on
+    one_qubit_gate_fidelities={"r": {0: 0.999, 1: 0.999, 2: 0.999, 3: 0.999, 4: 0.999}},
+    two_qubit_gate_fidelities={"cz": {(0, 2): 0.999, (1, 2): 0.999, (3, 2): 0.999, (4, 2): 0.999}},
+    # Gate fidelities for one and two qubit gates, the key in the dictionary indicates the qubits the gate operates on
```

Comment:
This is the exact same comment as the previous one. I think you don't even need these comments here since the names of variables are already self explanatory and the comment does not add anything new. Further info should be in the documentation, such as units, etc.